cli.js

```diff
@@ -3,6 +3,7 @@
 const meow = require('meow');
 const getStdin = require('get-stdin');
 const boxen = require('boxen');
+const leftPad = require('left-pad');
```

Comment:
Lol, no way :P

http://blog.npmjs.org/post/141577284765/kik-left-pad-and-npm

Can use https://github.com/sindresorhus/indent-string